2a_model/src/models/Snakefile_base.smk

```diff
@@ -128,22 +128,22 @@ def filter_predictions(all_preds_file, partition, out_file):
 
 
         if partition == "trn":
-            df_preds_filt = df_preds_trn_sites[(df_preds_trn_sites.date >= config['train_start_date'][0]) &
-                                               (df_preds_trn_sites.date < config['train_end_date'][0])]
+            df_preds_filt = df_preds_trn_sites[(df_preds_trn_sites.date >= config['train_start_date']) &
+                                               (df_preds_trn_sites.date < config['train_end_date'])]
         elif partition == "val":
             # get all of the data in the validation sites and in the validation period
             # this assumes that the test period follows the validation period which follows the train period
-            df_preds_filt_val = df_preds_val_sites[df_preds_val_sites.date < config['test_start_date'][0]]
```

Comment:
> TokenError in line 27 of .../drb-do-ml/2a_model/src/models/0_baseline_LSTM/Snakefile:
> ('EOF in multi-line statement', (229, 0))

Oh! I had deleted an extra "]" here when making these edits, and that's what was causing this python error 😭. I'll try re-running now.